### PR TITLE
Tips fragment feature

### PR DESCRIPTION
**Pull request by bryanporcello**
Pull request for the tips feature in the according fragment

### CODE REVIEW

File: app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt [hunk 1]
```diff
@@ -153,7 +153,19 @@ class MainActivity : AppCompatActivity() {
...
+                "It deals with the study of how organisms alter the environment for the benefit of themselves and other living beings. For eg, termites create a 6 feet tall mound and at the same time feed and protect their entire population.",
+                "Ecology plays a significant role in forming new species and modifying the existing ones. Natural selection is one of the many factors that influences evolutionary change.",
+                "Ecology was first devised by Ernst Haeckel, a German Zoologist. However, ecology has its origins in other sciences such as geology, biology, and evolution among others.",
+                "Habitat ecology is the type of natural environment in which a particular species of an organism live, characterized by both physical and biological features.",
+                "An organism free from the interference of other species and can use a full range of biotic and abiotic resources in which it can survive and reproduce is known as its fundamental niche."
+                )
+                replaceFragInMainContainer(AdviceFragment.newInstance(adviceList))
```
Comment (LucaCarroz) [inline app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt:168]: Those advices are very nice! 
I'm currently working on adding and retrieving data from firebase, if you want I can store those advices in the database and define a new function in `FirebaseDB` that retrieves the advices stored on the database: that way, we could update those advices anytime in the database and the users would see the update when they open this activity.
Tell me what you think about that :)

File: app/src/main/java/com/github/sdp_begreen/begreen/fragments/AdviceFragment.kt [hunk 2]
```diff
@@ -7,63 +7,48 @@ import android.view.ViewGroup
...
+
+        private const val QUOTES = "quotes"
         /**
          * Use this factory method to create a new instance of
          * this fragment using the provided parameters.
          *
-         * @param param1 Parameter 1.
-         * @param param2 Parameter 2.
-         * @return A new instance of fragment AdviceFragment.
+         * @param list list of quotes.
```
Comment (LucaCarroz) [inline app/src/main/java/com/github/sdp_begreen/begreen/fragments/AdviceFragment.kt:44]:
```suggestion
         * @param list of quotes.
```
little typo ;)

**Review by LucaCarroz — CHANGES_REQUESTED** [review]
Very nice addings, thanks for that!
I suggested a little modification concerning the way we retrieves the quotes, let me know what you think about it

File: app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt (hunk 1 above)
Comment (bryanporcello) [reply app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt:168]: Thanks for your comments and review Luca! Your suggestion definitely makes sense. I suggest to plan this refactoring task for the next sprint. What do you think?

**Review by LucaCarroz — APPROVED** [review]

File: app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt (hunk 1 above)
Comment (LucaCarroz) [reply app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt:168]: I just finished my second task and I have a bit of time left until friday so I can definitely add myself a new task to do that once your changes will be pushed to master!

Comment (bryanporcello) [reply app/src/main/java/com/github/sdp_begreen/begreen/activities/MainActivity.kt:168]: That's nice! Thanks for your hard work and devotion. I am obviously at your disposal may you need any help or detailed information regarding the code I wrote.

**Pull request merged**